verifier/client.go

```diff
@@ -39,9 +39,21 @@ type TokenOptions struct {
 type VerifyAttestationRequest struct {
 	Challenge                *Challenge
 	GcpCredentials           [][]byte
-	Attestation              *attestpb.Attestation
+	Attestation              *attestpb.Attestation // Attestation is for TPM attestations
 	ContainerImageSignatures []oci.Signature
 	TokenOptions             TokenOptions
+	TDCCELAttestation        *TDCCELAttestation // for TDX CCEL RTMR attestations
```

Comment:
Comments should go above the field, be capitalized, and end with a period.

Reply:
done